data/en/test_difference_in_years_month_year.json

```diff
@@ -0,0 +1,87 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "023",
+    "title": "Difference between two dates",
+    "description": "A test schema for calculate age from date",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "sections": [{
+        "id": "default-section",
+        "groups": [{
+            "id": "dates",
+            "title": "",
```

Comment:
Should not have empty titles in schemas (Applies to all other instances throughout the schemas)